doc/nrf/libraries/modem/modem_key_mgmt.rst

```diff
@@ -134,6 +134,17 @@ The following code snippet shows how to delete a CA chain certificate stored in
            printk("Failed to delete existing certificate, err %d\n", err);
    }
 
+The following code snippet shows how to delete all credentials associated with a security tag in the modem:
+
+.. code-block:: c
+
+   int err;
+
+   err = modem_key_mgmt_clear(<YOUR_SEC_TAG>);
```

Comment:
I would use an actual variable/constant here. Example:

```
.. code-block:: c

   int err;
   nrf_sec_tag_t sec_tag = 42;

   err = modem_key_mgmt_clear(sec_tag);
```

I prefer it because it makes the code copy-pasteable with an actual practical example.

Reply:
Added.